app/graphql/schema.graphql

```diff
@@ -102,6 +102,7 @@ type Profile implements Node & RulesPreload {
   majorOsVersion: String!
   name: String!
   policy: Profile
+  policyProfileId: ID
```

Comment:
Why are we exposing what is already exposed via 
```
policy {
  id
}
```

Reply:
Oh, sorry - I forgot that this was the profile ID :thinking: It's named poorly IMO if we are ever going to expose the actual policy object via graphql in the future (`initialProfileId` or similar would be more fitting). I added this field to both GraphQL and REST mostly for parity between the two profile endpoints, but I can remove it.

Reply:
Actually, I think it still makes sense as an attribute, since `policy` is `nil` for profiles without a policy, but we still have to report their policy type (even though it's the same as their name) in the UI. This attribute will allow us to not have any special frontend logic for displaying the policy type of external reports.

Reply:
Removed from GraphQL.